src/Core/src/Waiter.php

```diff
@@ -48,13 +48,6 @@ class Waiter
      */
     private $finalState;
 
-    /**
-     * A Result can be resolved many times. This variable contains the last resolve result.
-     *
-     * @var bool|NetworkException|null
-     */
-    private $resolveResult;
```

Comment:
This was never used. I guess we missed removing it when introducing the `Response` class